### DISCUSSION TITLE
Org name

### DISCUSSION THREAD

**Discussion by T1xx1** [abbrcode/abbreviations-in-code#88] [Polls]
[Announcement](https://github.com/abbrcodes/discussions/discussions/4)

**Comment by oood** [comment]
I just voted for `Abbreviate codes / Abbrcodes / Abbrcode`, in fact just `Abbrcode`

Because it's shorter, we don't need the plural, since that's an organization name, not a repository/project name, right?

@kisvegabor @philipeachille @T1xx1 

What do you think?

**Reply by T1xx1** [reply]
> I just voted for `Abbreviate codes / Abbrcodes / Abbrcode`, in fact just `Abbrcode`

Ok

> Because it's shorter, we don't need the plural, since that's an organization name, not a repository/project name, right?

Yes. It's the name of the org.

> @kisvegabor @philipeachille @T1xx1 
> 
> What do you think?
> 

I voted AbbreviationsInCode as the actual repo called abbreviations-in-code will be renamed dictionary after the dictionary project

**Comment by kisvegabor** [comment]
Actually `abbrcode`, `Code-abbrev `, `AbbreviationsInCode / abbreviations-in-code` are equally good for me. :slightly_smiling_face:

**Reply by oood** [reply]
> Actually `abbrcode`, `Code-abbrev `, `AbbreviationsInCode / abbreviations-in-code` are equally good for me. 🙂


For me: 

org: `abbrcode`

repository/project `abbreviations-in-code`

**Reply by T1xx1** [reply]
For me

Org: AbbreviationsInCode
Repo: Dictionary

**Reply by oood** [reply]
But there may also be a second dictionary, like acronyms-in-code.

**Reply by T1xx1** [reply]
> But there may also be a second dictionary, like acronyms-in-code.

But as @philipeachille said has to be different so we would not call it dictionary again.

**Reply by oood** [reply]
🤔

**Reply by T1xx1** [reply]
In my opinion acronyms are abbrs so it makes sense to call the org AbbreviationsInCode.

**Reply by oood** [reply]
~~Yes, they are close, but abbreviations refer to words, while acronyms can include phrases.~~


Checked the [dictionary](https://www.dictionary.com/e/acronym-vs-abbreviation/) and you were right.

> An abbreviation is **any** shortened or contracted form of a _word_ or _**phrase**_.

**Reply by T1xx1** [reply]
![Screenshot (116)](https://user-images.githubusercontent.com/69322987/224116959-074224ee-2f33-4573-9b50-89286443f52a.png)

That's my visual representation of our thing.

**Reply by oood** [reply]
it's very clear

**Comment by philipeachille** [comment]
Thinking about it for a while I now think "abbreviations-in-code" is a good org name and the repo can either be called the same or ust "abbreviations". I think I also prefer "abbreviations-in-code" for the repo-name

**Reply by oood** [reply]
Yep, I like that the repository has that name, that's how I found it.   But don't you think as an organization, it's too long?

**Reply by T1xx1** [reply]
Ok for the repo name, can we just capitalize it?

How about AbbrInCode as org name so it's similar but shorter.

**Reply by oood** [reply]
maybe just AbbrCode?

**Reply by T1xx1** [reply]
Deal. Should we change also the username?

**Reply by oood** [reply]
I'm confused, I mean the username should be this `AbbrCode`.

**Reply by oood** [reply]
Because some people think that plurals are used in inappropriate places (`abbrcodes` or `abbrscode`), I suggest not to use plurals at all (`abbrcode`).

**Reply by T1xx1** [reply]
Username and org name are 2 different things. The username is the name in front of the repo name. The name is a random name you can give to your org.

**Reply by philipeachille** [reply]
I settle for this: https://github.com/abbrcode/abbreviations-in-code

**Reply by T1xx1** [reply]
Okay, done.

**Reply by oood** [reply]
Great!
